app/views/users/show.html.erb

```diff
@@ -1 +1,44 @@
-<h1>This is the user profile</h1>
\ No newline at end of file
+<head>
+  <link rel="stylesheet" href="/assets/stylesheets/users_show.css">
+</head>
+<body>
+  <h1>This is the user profile</h1>
+    <div class="usersShow-container">
+      <div class="userShow-header">
+        <div class="userShow-image">
+          <img @user.photo alt="User Image">
+        </div>
+        <div class="userShow-details">
+          <span id="userName"><%= @user.name %></span>
+          <div id="userPostContainer">
+            <span class="user-post-count">Number of posts: <%= @user.posts.count %></span>
+          </div>
+        </div>
+      </div>
+      <div class="user-bio">
+        <span>Bio</span>
+        <p><%= @user.bio %></p>
+      </div>
+      <div class="user-posts-container">
+        <% @user.three_most_recent_post.each do |post| %>
+          <div class="post-item-container">
+            <span class="post-title"><%= post.title %></span>
+            <p class="post-text"><%= post.text %></p>
+            <div class="post-details">
+              <span class="post-detail">Comments:
+                <%= post.comments.count %>
+              </span>
```

Comment:
- [ ] I'm not able to add comments to your app. The addition of `users` and `posts` was a success, but adding `comments` was an issue. 
- On looking at the error, i noticed its from your `comment` model where you did not reference the `Author_id` as it was stated in the [ERD](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_app_erd.png) wireframe given.

### Error prompt:
![image](https://github.com/ivangonzalez224/Blog/assets/50990925/444870ab-acf5-42e9-9750-6959d3517682)


### Model view:
![image](https://github.com/ivangonzalez224/Blog/assets/50990925/ea243ae4-62c8-4a6d-8ec7-24c78a0fb539)

### Your model:
![image](https://github.com/ivangonzalez224/Blog/assets/50990925/09ddfbae-d5b0-4dee-b46c-b100508422e3)

Reply:
Hi @Shedrack-Sunday thank you for the review.
You got an error when creating a new comment because you added the wrong attribute (author_id), you should add the 'user' attribute because you need to add a user object and not a user id as an attribute, the same way you added the 'post' attribute with the first_post object and you did not use the 'post_id' as an attribute.
The correct way to create a comment is as follows:
Comment.create(post: first_post, user: second_user, text: 'Hi Tom' )

Reply:
Hello @ivangonzalez224 , I added the `author_id` because the project was not initially running based on the migration from your database which has `author` instead. That's why I made the suggestion. Of course, it would have run by adding post_id, but that was not what was initially in your migration which was why i asked for changes